### ISSUE TITLE
Improve styling/theming of NextTable and SortableHeader

### ISSUE THREAD

**Issue by tealjulia** [airbytehq/airbyte#23684]
When working on https://github.com/airbytehq/airbyte-platform-internal/pull/4792 there were some properties that needed to be overridden in order to fully implement the "light" theme of this table to match the designs. 

We should:
- Fully implement the "light" theme such that custom overrides are not required
- support highlighting when a column is the one actively sorted in SortableHeader

**Comment by flash1293** [comment]
@timroes @tealjulia Is this done already?

**Comment by flash1293** [comment]
@timroes @tealjulia Is this done already?

**Comment by tealjulia** [comment]
@flash1293 No.  It is better, but there are some manual SCSS overrides of the theme in the usage table on the billing page.  I am happy to take this on, though.

**Comment by tealjulia** [comment]
Thanks.  I'm on it.  Waiting for https://github.com/airbytehq/airbyte-platform-internal/pull/5169 and https://github.com/airbytehq/airbyte-platform-internal/pull/5286 to be merged but should get to it this sprint!
